specs/executionAPISpec.md

```diff
@@ -0,0 +1,530 @@
+* [EFH-3] eth_feeHistory **MUST** allow block tags to be used for `highestBlock`.
+* [EFH-4] eth_feeHistory **MUST** error with code -32000 when `highestBlock` is ahead of the chain.
+## <p id="blockNumber">eth_blockNumber</p>
+* [EBN-1] eth_blockNumber **MUST** return the block number for the most recent block mined.
+* [EBN-2] eth_blockNumber **MUST** return "0x0" when the client is not synced to the network.
+## <p id="sign">eth_sign</p>
+* [ESN-1] eth_sign **MUST** return the Ethereum specific signature detailed in [EIP-191](https://eips.ethereum.org/EIPS/eip-191) for the given unlocked `address` and `message`.
+* [ESN-2] eth_sign **MUST** error with code -32000 when the account corresponding to the `address` is not unlocked.
+* [ESN-3] eth_sign **MUST** error with code -32000 when the account corresponding to the `address` is not owned by the client.
+## <p id="call">eth_call</p>
+* [EC-1] eth_call **MUST** return the result of the given transaction.
+* [EC-2] eth_call **MUST** accept all current transaction types. Legacy transactions and [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) "typed" transactions.
+* [EC-3] eth_call **MUST NOT** mine any transaction on the blockchain.
+* [EC-4] eth_call **MUST** use the block requested by the `defaultBlockParameter` when interacting with contracts.
+* [EC-5] eth_call **MUST** error with code -32000 when the requested block does not exist or is not available.
+* [EC-6] eth_call **SHOULD NOT** be allowed to be called from an address where CODEHASH != EMPTYCODEHASH. [EIP-3607](https://eips.ethereum.org/EIPS/eip-3607)
+* [EC-7] eth_call **MUST NOT** error if the transaction exceeds the gas or fee cap.
+* [EC-8] eth_call **MUST** use 0x0000000000000000000000000000000000000000 as default `from` address when `from` is null or not specified. [Nethermind uses 0xf...fe] 
+* [EC-9] eth_call **MUST** check `from` account balance has sufficient funds to "pay" for the transaction.
+* [EC-10] eth_call **MUST** error with code -32000 account has insufficient funds to execute the transaction call.
+* [EC-11] eth_call **MUST NOT** calculate cost of transactions when the `gas` and a price per unit of gas is not specified.   
+* [EC-12] eth_call **MUST** allow `data` or `input` to be used when testing deployment or interacting with contracts.
+* [EC-13] eth_call **MUST** use `input` when both `input` and `data` are specified. 
+* [EC-14] eth_call **MUST** error with code -32000 when the `data` or `input` being deployed causes and error in the EVM.
+* [EC-15] eth_call **MUST** error with code -32000 when the `gas` is too low to execute the call.
+* [EC-16] eth_call **MUST** error with code -32000 when using `gasPrice` with `maxFeePerGas` or `maxPriorityFeePerGas`. 
+* [EC-17] eth_call **MUST** use networks chain id for `chainId`
+* [EC-18] eth_call **MUST** error with code -32000 when using block number or block hash for `defaultBlockParameter` while syncing to the network.
```

Comment:
This is incorrect.  While it is common for a syncing node to not be able to serve requests and thus will error, this is not a requirement that should be specified.  One can imagine a client that *can* serve requests while syncing.

Also note: Clients are *always* syncing.  I think what you mean here is "during initial sync", but the above comment still applies that this shouldn't be specified.